app/1.0/docs/devguide/local-dom.md

```diff
@@ -10,7 +10,7 @@ its _light DOM_ for clarity.
 
 Polymer supports multiple local DOM implementations. On browsers
 that support shadow DOM, shadow DOM may be used to create local DOM. On other
-supported browsers, Polymer provides local DOM via a custom
+not supported browsers, Polymer provides local DOM via a custom
```

Comment:
"Supported" could mean "supported by Polymer" or "that support shadow DOM". I think the original intention was probably the first one, but it's quite confusing. I think a better fix would be to remove the word "Supported" entirely, "On other browsers ..."